### PR TITLE
DOCS-3097: Fix missing sessions options snippets

### PR DESCRIPTION
**Pull request by JessamyT**

### CODE REVIEW

File: docs/appendix/apis/sessions.md [hunk 1]
```diff
@@ -125,9 +125,46 @@ The `SessionsClient` that serves the session management API is automatically ena
...
+
+For more information, see the [TypeScript SDK Docs](https://ts.viam.dev/interfaces/DialWebRTCConf.html).
+
+{{% /tab %}}
+{{% /tabs %}}
+
 This option allows you to have full control over sessions management.
 After disabling the client, you must now manage each of your sessions manually with the session management API.
-You can do this with Viam's [client SDKs](https://pkg.go.dev/go.viam.com/rdk/session).
+You can do this with Viam's client SDK.
```
Comment (npentrel) [inline docs/appendix/apis/sessions.md:167]:
```suggestion
You can do this with Viam's client SDKs.
```
You can do it with multiple

File: docs/appendix/apis/robot.md [hunk 2]
```diff
@@ -39,7 +39,6 @@ from viam.robot.client import RobotClient
 
 async def connect() -> RobotClient:
     opts = RobotClient.Options(
-        disable_sessions=True,
         dial_options=DialOptions(timeout=10)).with_api_key(
```
Comment (npentrel) [inline docs/appendix/apis/robot.md:42]: This Also shouldn't be there I think

**Review by npentrel — APPROVED** [review]

File: docs/appendix/apis/robot.md (hunk 2 above)
Comment (JessamyT) [reply docs/appendix/apis/robot.md:42]: Which part? the timeout?

Comment (npentrel) [reply docs/appendix/apis/robot.md:42]: Yes

Comment (JessamyT) [reply docs/appendix/apis/robot.md:42]:
```suggestion
        dial_options=DialOptions.with_api_key(
```

File: docs/appendix/apis/sessions.md [hunk 3]
```diff
@@ -125,9 +125,46 @@ The `SessionsClient` that serves the session management API is automatically ena
...
+    robot = await connect()
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient.Options.disable_sessions).
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+```go {class="line-numbers linkable-line-numbers"}
+robot, err := client.New(ctx, "my-machine-address", logger, client.WithDisableSessions(), ...)
```
Comment (JessamyT) [inline docs/appendix/apis/sessions.md:144]:
```suggestion
robot, err := client.New(
  ctx, "my-machine-address",
  logger,
  client.WithDisableSessions(),
  ...)
```

**Comment by JessamyT** [comment]
merged per offline approval from Felix

**Pull request merged**